testing/integration-neo4j/src/test/java/com/neueda/jetbrains/plugin/graphdb/test/integration/neo4j/tests/cypher/parameters/CypherParametersProviderTest.java

```diff
@@ -38,19 +48,40 @@ public void setUp() throws Exception {
     }
 
     public void testParsingEmptyJsonObject() throws Exception {
+        SettingsComponent.getInstance().enableFileSpecificParams(false);
         parametersProvider.setParametersJson("{}");
+        parametersProvider.setLocalParametersJson("{\"param\": \"non-empty\"}");
+        Map<String, Object> parameters = parametersService.getParameters(getPsiFile("RETURN $param"));
+        assertThat(parameters).isEmpty();
+    }
+
+    public void testParsingEmptyJsonObjectInLocalParams() throws Exception {
+        SettingsComponent.getInstance().enableFileSpecificParams(true);
+        parametersProvider.setLocalParametersJson("{}");
+        parametersProvider.setParametersJson("{\"param\": \"non-empty\"}");
```

Comment:
Order of lines differ from test above

Reply:
fixed in [d787900e](https://github.com/ex3m1024/jetbrains-plugin-graph-database-support/commit/d787900e73bc18d48e9b0ac186e839f178cb809b)